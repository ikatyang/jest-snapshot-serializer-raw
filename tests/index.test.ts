import { expect, test } from 'vitest'
import serializer, { wrap } from '../src/index.js'

expect.addSnapshotSerializer(serializer)

const example = `paragraph "one"\n\n'paragraph' \\two\\`

test('before', () => {
  expect(example).toMatchSnapshot()
})

test('after', () => {
  expect(wrap(example)).toMatchSnapshot()
})
