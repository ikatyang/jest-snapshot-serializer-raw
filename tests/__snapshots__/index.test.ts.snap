// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`after 1`] = `
paragraph "one"

'paragraph' \\two\\
`;

exports[`before 1`] = `
"paragraph \\"one\\"

'paragraph' \\\\two\\\\"
`;
